configure.ac

```diff
@@ -2474,7 +2501,7 @@ SURICATA_BUILD_CONF="Suricata Configuration:
   PCRE jit:                                ${pcre_jit_available}
   LUA support:                             ${enable_lua}
   libluajit:                               ${enable_luajit}
-  GeoIP2 support:                          ${enable_geoip}
+  GeoIP support:                           ${enable_geoip}
```

Comment:
instead of just yes/no it would be good to list the flavor in use. E.g. "yes, libgeoip" or "yes, libmaxmindb"